app/furniture/marketplace/delivery/window.rb

```diff
@@ -0,0 +1,29 @@
+class Marketplace
+  class Delivery
+    class Window < Model
+      attr_accessor :value
+      delegate(:nil?, :blank?, :empty?, to: :value)
+
+      def strftime(*args, **kwargs)
+        return value.strftime(*args, **kwargs) if time_like?
+        value
```

Comment:
Callers of `strftime` will probably expect a string coming back from this call. Is `value` a string? Or can it be something else that should be cast to a string?

Reply:
Oo that's a good point... I could just call `to_s`? For now, `value` is either a string or a `DateTime`; but if we start wanting it to hold a date range or other less primitive data structures it will be weird to return them.